server/settings.py

```diff
@@ -87,6 +87,12 @@ def env(variable, fallback_value=None):
     "copyrightnotice": {
         "required": False,
     },
+    "copyrightholder": {
+        "required": True,
+    },
+    "usageterms": {
```

Comment:
those fields are already in dict with different values

Reply:
yep I just saw it in travis, already fixed